shared/actions/settings.tsx

```diff
@@ -416,9 +418,14 @@ const editEmail = (_, action: SettingsGen.EditEmailPayload, logger: Saga.SagaLog
     return RPCTypes.emailsSetPrimaryEmailRpcPromise({email: action.payload.email})
   }
   if (action.payload.verify) {
-    return RPCTypes.emailsSendVerificationEmailRpcPromise({email: action.payload.email}).then(() =>
-      SettingsGen.createSentVerificationEmail({email: action.payload.email})
-    )
+    return RPCTypes.emailsSendVerificationEmailRpcPromise({email: action.payload.email}).then(() => [
+      SettingsGen.createSentVerificationEmail({email: action.payload.email}),
+      PeopleGen.createGetPeopleData({
```

Comment:
I don't think that's the right way to do it. It won't update other clients that may be observing people tab.

With the last fix to propagate todo updates using gregor state notifications, we should be fine if the server sends the update notification. But I'm not 100% familiar with how this works right now, I'll find out.